src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -40,18 +38,14 @@ export const getInstallations = async (client: GitHubAPI, subscriptions: Subscri
 	};
 };
 
-const getInstallation = async (client: GitHubAPI, subscription: Subscription, log: Logger): Promise<AppInstallation> => {
+const getInstallation = async (subscription: Subscription, log: Logger): Promise<AppInstallation> => {
 
 	const { jiraHost } = subscription;
-	const useNewGitHubClient = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_FOR_GET_INSTALLATION, false, jiraHost) ;
```

Comment:
Shouldn't the flag also be removed from feature-flags file?

Reply:
oo thank you, good catch!